src/views/SpaceCreate.vue

```diff
@@ -493,7 +493,8 @@ onBeforeRouteLeave(async () => {
             web3.authLoading ||
             hasAuthorValidationFailed ||
             validationLoading ||
-            isGnosisAndNotSpaceNetwork
+            isGnosisAndNotSpaceNetwork ||
+            space.hibernated
```

Comment:
Button should only be disabled for non admins/controller

Reply:
Why disabled only for non-admin/controller, if proposal creation is disabled for everyone ?